spk/mono/Makefile

```diff
@@ -36,3 +36,5 @@ mono_extra_install:
 mono_pre_copy: $(INSTALL_PLIST)
 	# remove boehm GC references from PLIST if it isn't available on the platform (e.g. rtd1296)
```

Comment:
@m4tt075 Would it be possible to use recent per-cross PLIST_TRANSFORM with per-arch condition? I guess not only rtd1296 is concerned by "mono-boehm" but all ARM8 so aarch64 virtual arch.